lib/utils/middlewares.js

```diff
@@ -0,0 +1,34 @@
+import {getContext} from '../context'
+
+export const buildContext = async (argv) => {
+  const {
+    managementToken,
+    cmaToken,
```

Comment:
I think we should align all arguments to be `managementToken` instead of having 2 property for the same value

Reply:
Good idea, i changed `cmaToken` to `managementToken` and `cdaToken` to `deliveryToken` to have a consistent naming